libkbfs/favorites.go

```diff
@@ -142,6 +160,7 @@ func (f *Favorites) loop() {
 
 // Shutdown shuts down this Favorites instance.
 func (f *Favorites) Shutdown() {
+	atomic.StoreInt32(&f.shutdown, 1)
```

Comment:
I've never used `atomic` before, and the documentation says it probably shouldn't be used by high-level code.  I agree this all looks ok, but maybe we should use a `sync.Mutex` and a `bool` to be consistent with the documentation and our other code?  I don't feel too strongly about it though.